src/ucs/debug/profile.h

```diff
@@ -366,20 +366,32 @@ static inline void ucs_profile_record(ucs_profile_type_t type, const char *name,
 
 
 /*
- * Profile a function call which does not return a value..
+ * Profile a function call which does not return a value, and specify explicit
+ * name string for the profile. Useful when calling a function by a pointer.
  *
+ * @param _name   Name string for the profile.
  * @param _func   Function name.
  * @param ...     Function call arguments.
  */
-#define UCS_PROFILE_CALL_VOID(_func, ...) \
+#define UCS_PROFILE_NAMED_CALL_VOID(_name, _func, ...) \
     { \
         UCS_PROFILE_SCOPE_BEGIN(); \
         _func(__VA_ARGS__); \
-        UCS_PROFILE_SCOPE_END(#_func); \
+        UCS_PROFILE_SCOPE_END(_name); \
     }
 
 
 /*
+ * Profile a function call which does not return a value..
+ *
+ * @param _func   Function name.
+ * @param ...     Function call arguments.
+ */
+#define UCS_PROFILE_CALL_VOID(_func, ...) \
```

Comment:
Description of this macros is really important. I would suggest to add examples how function with arguments gets translated to this macro